aws/adapter.go

```diff
@@ -660,6 +660,60 @@ func (a *Adapter) FindManagedStacks(ctx context.Context) ([]*Stack, error) {
 	return stacks, nil
 }
 
+// GetStackELBs returns the list of load balancers of the managed stacks.
+// While implementing this method it was taken that each stack has its own load
+// balancer and never refer to the same load balancer from multiple stacks.
+//
+// If a stack does not have a load balancer ARN it will be returned with an empty
+// ELB field.
+func (a *Adapter) GetStackELBs(ctx context.Context, stacks []*Stack) ([]*StackELB, error) {
+
+	lbARNs := make([]string, 0, len(stacks))
+	for _, stack := range stacks {
+		if stack == nil || stack.LoadBalancerARN == "" {
+			continue
+		}
+		lbARNs = append(lbARNs, stack.LoadBalancerARN)
+	}
+
+	lbsMap := make(map[string]*elbv2Types.LoadBalancer, len(lbARNs))
+
+	paginator := elbv2.NewDescribeLoadBalancersPaginator(
+		a.elbv2,
+		&elbv2.DescribeLoadBalancersInput{
+			LoadBalancerArns: lbARNs,
```

Comment:
What happens if we have more than 20 ARNs?

Reply:
Paginator takes care of it if I'm not mistaken. This is the way it was done with the stacks retrieval, I guess the same limitation applies there as well right?

Reply:
I'll double check this anyways

Reply:
[AWS API reference of DescribeLoadBalancers](https://docs.aws.amazon.com/elasticloadbalancing/latest/APIReference/API_DescribeLoadBalancers.html)

Reply:
Accommodated 20 loadBalancerARNs request param limit in DescribeLoadBalancers API with the [getLoadBalancerStates function](https://github.com/zalando-incubator/kube-ingress-aws-controller/pull/748/files#diff-0efc4c5374da02ee834edf241846dcc4400c726b8b09af68804abef6a2f126d8R77-R119)